### PR TITLE
AG edit profile Display subject list

### PR DESCRIPTION
**Pull request by Dhanashree-Patil95**

### CODE REVIEW

File: apps/front-end/src/pages/front-end/BenificiaryEnrollment.js [hunk 1]
```diff
@@ -155,25 +155,16 @@ export default function BenificiaryEnrollment() {
...
-                          <React.Fragment>
-                            {e?.name}
-                            {"\n"}
-                          </React.Fragment>
-                        );
-                      }
-                    })}
+                    }
+                  })}
                 </FrontEndTypo.H3>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/BenificiaryEnrollment.js:168]: change as per discussed .

File: apps/front-end/src/pages/front-end/BenificiaryProfileView.js [hunk 2]
```diff
@@ -107,7 +105,7 @@ export default function BenificiaryProfileView(props) {
               {benificiary?.first_name}
               {benificiary?.last_name && ` ${benificiary?.last_name}`}
             </FrontEndTypo.H2>
-            <Box>{benificiary?.status || "unidentified"}</Box>
+            <Box>{benificiary?.status || "identified"}</Box>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/BenificiaryProfileView.js:108]: use status button.

File: apps/front-end/src/pages/front-end/BenificiaryEnrollment.js (hunk 1 above)
Comment (Dhanashree-Patil95) [reply apps/front-end/src/pages/front-end/BenificiaryEnrollment.js]: Resolved

File: apps/front-end/src/pages/front-end/BenificiaryProfileView.js (hunk 2 above)
Comment (Dhanashree-Patil95) [reply apps/front-end/src/pages/front-end/BenificiaryProfileView.js]: Resolved

**Pull request merged**